connect/telegram_bot/telegram_conversation.py

```diff
@@ -220,18 +212,14 @@ def _end_conversation(self, update):
         return ConversationHandler.END
 
     def execute_custom_command(self, update, command):
-        old_stdout = sys.stdout
-        sys.stdout = mystdout = StringIO()
-
+        out = StringIO()
         try:
-            execute_from_command_line(['', command])
-            self._reply(update, f'Result {command}:\n{mystdout.getvalue()}')
+            call_command(command, strerr=out, stdout=out)
```

Comment:
Don't understand where if control that the command received from the telegram conversation is one of the declared available.
As we told, probably is not a good idea to expose all the available commands through the conversations (ex: createsuperuser).

Reply:
added such check